services/web/client/source/class/osparc/component/export/Permissions.js

```diff
@@ -31,6 +31,10 @@ qx.Class.define("osparc.component.export.Permissions", {
 
     this.__buildLayout();
 
+    this.__collaborators = {};
+    initCollabs.forEach(initCollab => {
+      this.__collaborators[initCollab["gid"]] = initCollab;
```

Comment:
when you share with everyone? Do you consider the product?
I.e. if a user shares with everyone in osparc, I assume everyone using osparc should get the service
but if he/she shares it in s4l product, then I assume "everyone" refers to users USING s4l specifically

Reply:
@sanderegg

Reply:
As explained earlier: currently the product constraint is applied through the services access rights.

So:
Everyone is everyone regardless of the products. BUT, the services are constrained by products, so indirectly if you share a study with everyone while browsing on osparc, and that study uses services not available in s4l product, then that study will not appear in s4l.